qucs/qucs/octave_window.cpp

```diff
@@ -117,7 +117,7 @@ bool OctaveWindow::startOctave()
   octProcess.waitForStarted();
 
   if(octProcess.state()!=QProcess::Running) {
-    output->setText(tr("ERROR: Failed to execute " + Program));
+    output->setText(tr("ERROR: Failed to execute ") + Program);
     return false;
```

Comment:
well, the user may just get a ungrammatical error message, for some languages; the Qucs GUI will still do its main purpose, of course. You can call it an issue, instead of a bug :grin:
In any case, I guess there are similar "issues" in other parts of the code around, we could as well open a dedicated issue to track that.